src/AllPolls.js

```diff
@@ -0,0 +1,81 @@
+import { Link } from "react-router-dom";
+import firebase from "./firebase";
+import { getDatabase, onValue, ref} from "firebase/database";
+import { useEffect, useState } from "react";
+import Header from "./Header";
+
+const AllPolls = () => {
```

Comment:
this is a fun app! the puns make me smile. the logic of your route paths are organized so well. perhaps you can apply flexbox to reverse your AllPolls page so that newer polls that are submitted are on the top instead of scrolling to the bottom?

Reply:
done!